arch/arm64/boot/dts/qcom/sm7150-xiaomi-tucana.dts

```diff
@@ -138,6 +225,42 @@
 	enable-gpios = <&tlmm 67 GPIO_ACTIVE_HIGH>;
 };
 
+&panel {
+	compatible = "xiaomi,f4-41-06-0a";
+};
+
+&pm6150l_flash {
+	status = "okay";
+
+	/* Both LEDs */
+	led-1 {
```

Comment:
Actually this is not correct. We should take care that the LEDs in kernel space are declared correctly. And userspace should decide which LEDs to turn on.

So drop this subnode.